src/twisted/conch/ssh/keys.py

```diff
@@ -1093,59 +1094,59 @@ def size(self):
             return 256
         return self._keyObject.key_size
 
-    def data(self):
+    def data(self) -> dict:
         """
         Return the values of the public key as a dictionary.
 
         @rtype: L{dict}
         """
         if isinstance(self._keyObject, rsa.RSAPublicKey):
-            numbers = self._keyObject.public_numbers()
```

Comment:
mypy is really annoying here demanding a new numbers variable for every different type of public/private numbers

Reply:
I can understand why it requires separate names.

Does it still contain, even when reqriting  to code not to use `elif`... like ?

```
if condition:
  return

if other-condition:
  return
```

Reply:
yep I tried without elif first